lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -137,6 +137,22 @@ def initialize(model, current_user:, embed_links: false)
           }
         end
 
+        link :'custom-fields' do
+          {
+            href: 'custom-fields',
```

Comment:
I'd prefer If we pass the correct path here instead of putting it together in the frontend.